docs/features/repo-pages.md

```diff
@@ -0,0 +1,47 @@
+---
+id: repo-pages
+title: "Check out Repository Activity"
+sidebar_label: "Repository Pages"
+keywords:
+  - "repository pages"
```

Comment:
Do we want to keep consistency by using capitalization across docs for keywords? Or how would it be?

```suggestion
  - "Repository Pages"
```